### ISSUE TITLE
[Bug] Extensive Tweaking of the Config Breaks Feather Regen and Dodge Cost

### ISSUE THREAD

**Issue by NebelFox** [ElenaiDev/ElenaiDodge2.0#30]
Minecraft version: 1.12.2
Mode version: 1.0.8b

I launched first time with this mod. Tested dodge. Everything seemed fine. Then I decided to tweak some values in config. Changed the force, some hud options, and feather replenishment rate as well as dodge cost. After that I was unable to dodge at all, after I had use all my feathers. They just weren't regenerating and the first one was obfuscated. Resetting the config to default values didn't help.
Now I'm installing the 1.0.9b version of the mod to check how it works. Will update this post after some tests.

Have similar issue on 1.0.9b. I even didn't changed the config. Just entered the game, turned gamemode 0, and tried to dash. The first single dash emptied my feathers bar, and they weren't refilling. So I was unable to dodge again.

**Comment by ElenaiDev** [comment]
That's unusual, sorry for the inconvenience! Have you tried this on other worlds?

**Comment by NebelFox** [comment]
No, I haven't. If you imply that the problem might be in the save, I also incline to this thought. However, It needs to be tested. I'm gonna do this when I get some free time, and then will inform you about results.

**Comment by ElenaiDev** [comment]
I have just changed the title to help me keep track of the exact issue.
Would you be able to send me your config file to have a look at? Something may have gone wrong.

Thank you! :)

**Comment by NebelFox** [comment]
I'm so sorry I disappeared for those 2 weeks. 
And unfortunately i can't provide the config file as I deleted it to test another version where I haven't changed anything
In the first config I definitely changed the following values:
balance.hunger -> 14
feathers.cost -> 6
feathers.regen_rate -> 100

 Most likely it's a problem with feathers.regen_rate, as my feathers didn't replenish after I used them all.

**Comment by ElenaiDev** [comment]
No need to apologize!

I've been out of action for a while too due to exams, so I'm only just able to get things back on track.

That's very unusual that feathers won't regenerate. I'll do some testing to see if I can fix it!
